mixer/adapter/bypass/config/bypass.yaml

```diff
@@ -1,5 +1,5 @@
 # this config is created through command
-# mixgen adapter -c $GOPATH/src/istio.io/istio/mixer/adapter/bypass/config/config.proto_descriptor -o $GOPATH/src/istio.io/istio/mixer/adapter/bypass/config -n bypass -t checknothing -t reportnothing -t metric -t quota
+# mixgen adapter -c /work/mixer/adapter/bypass/config/config.proto_descriptor -o /work/mixer/adapter/bypass/config -n bypass -t checknothing -t reportnothing -t metric -t quota
```

Comment:
did you mean for this to be hardcorded to `/work/` here (instead of `$REPO_ROOT`) ?

Reply:
It's just dumping the resolved path there. I'm not sure how come it used to show up as $GOPATH. Do you think this is significant?

Reply:
It's fixed, the strings now say $REPO_ROOT/...